app/models/product.rb

```diff
@@ -1,4 +1,21 @@
 class Product < ApplicationRecord
+  PRODUCT_TAX_RATE = 0.08
```

Comment:
```suggestion
  TAX_RATE = 0.08
```
`Product::TAX_RATE`なのだからPRODUCTはいらないんでは